02.create-and-explore/03.start-working-in-your-sandbox/14.creating-and-securing-your-domain/docs.md

```diff
@@ -0,0 +1,154 @@
+    *If you also downloaded Interface with Sandbox, you will automatically start in your domain*  
+    ![localhost-search.png "Localhost"]
+3. Select the “Create” icon from the toolbar to view various options to add content to your domain  
+    ![create-main-menu.png "Create Menu"]
+4. Let’s add a “primitive” - a simple 3D object that can be used to build larger structures  
+    ![primitive.png "primitive"]
+5. There are many properties that can be modified in an entity, such as:
+    - color
+    - name
+    - shape type
+    - interactivity (e.g. grabbable, triggerable, cloneable)
+    - scripting behavior  
+	*All of these options are available in the “Properties” submenu when an entity is selected*  
+ 	![primitive-properties.png "Primitive Properties"]
+6. Switch to the “List” submenu to search for your newly created entity using the filter bar  
+    - You can disable editing, deleting, or moving the entity for everyone by choosing the lock icon from this menu or the “Properties” submenu
+    - Unlock it to reenable editing  
+    ![list.png "List"]
+
+## Adjusting your Domain Server Settings
+
+1. Sandbox settings are accessible via browser. You can get to them by right-clicking your hidden icons list or by going to [your server settings URL](http://localhost:40100/settings/)  
+	![hidden-icons.png "Sandbox Icon"]
+2. Open your domain server settings by right-clicking on this icon  
+	![sandbox-icon-right-click.png "Sandbox Icon Right Click"]
+3. The following screen will appear in your default browser  
+	![settings-top.png "Server Settings"]
+4. In Domain Settings, you can:
+	- update place names (i.e. the metaverse equivalent of a URL)
+	- add a description of your domain
+	- update your security settings
+	- access or update your content archives
+5. Update your username and password here  
+	*This can be a unique name, and not necessarily your High Fidelity username*  
+	![domain-security.png "Domain Security"]
+6. Update the standard permissions to specify who can come in and change your domain  
+	![standard-permissions.png "Standard Permissions"]
+7. You can also grant permissions to specific users under the "Permissions for Specific Users" table by inputting their usernames  
+8. Types of users:  
+	![user-permissions.png "User Permissions"]
+	- An **anonymous** user is a person who is not logged in and is using an instance of High Fidelity's Interface (the "client")
+		- This is the default user type for someone who has downloaded Interface for the first time
+	- A **friends** user is someone that you have connected with by shaking hands with their avatar and clicking on the "Friends" checkbox in your "People" app  
+		- **Connections** are different than **friends**. Friends are highlighted in green and by default have more rights in your domains than connections highlighted in purple (who are treated like anonymous users in your server)  
```

Comment:
"...connections highlighted in purple that are treated as anonymous users in your server "